man/assign.Rd

```diff
@@ -92,12 +92,12 @@ Since \code{[.data.table} incurs overhead to check the existence and type of arg
 \value{
 \code{DT} is modified by reference and returned invisibly. If you require a copy, take a \code{\link{copy}} first (using \code{DT2 = copy(DT)}).
 }
-\seealso{ \code{\link{data.table}}, \code{\link{copy}}, \code{\link{setalloccol}}, \code{\link{truelength}}, \code{\link{set}}, \code{\link{.Last.updated}}
+\seealso{ \code{\link{data.table}}, \code{\link{copy}}, \code{\link{setalloccol}}, \code{\link{truelength}}, \code{\link{set}}, \code{\link{.Last.updated}},\code{\link{internal.selfref}}
```

Comment:
When linking to a help page, [use the `\alias{...}` name of the help page (i.e. `.internal.selfref` here), not its file name (i.e. not `internal.selfref`)](https://cran.r-project.org/doc/manuals/R-exts.html#Cross_002dreferences). All other occurrences will need to be fixed too.